internal/gengapic/testdata/doc_file.want

```diff
@@ -16,6 +16,10 @@
 
 // Package awesome is an auto-generated package for the
 // Awesome Foo API.
+
```

Comment:
I think this was a bug in the previous generator. We shouldn't have new newline.

Reply:
Looks like we still have a new line here though?

Reply:
Oh i thought you meant the empty new line comment that was immediately below it.

Should it be:

```
// Package awesome is an auto-generated package for the
// Awesome Foo API.
//
// Documentation description goes here
package foo
```

or

```
// Package awesome is an auto-generated package for the
// Awesome Foo API.
// Documentation description goes here
package foo
```

or

```
// Package awesome is an auto-generated package for the
// Awesome Foo API.

// Documentation description goes here
package foo
```

Reply:
I think it should be the first. The `//` line makes `godoc` renders "Package awesome..." and "Documentation..." on different paragraphs.

The second one will render them in the same paragraph, which works but I think not as good.
The third will render only "Documentation.." because "Package awesome..." will become "orphan" I think.